packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -200,11 +211,10 @@ <h2>
                                     <option
                                         ng-repeat="perm in $ctrl.volumesPermissions"
                                         value="{{perm}}"
-                                    >{{::
+                                        >{{::
                                         'pci_projects_project_training_jobs_submit_data_permission_'
-                                    + perm | translate }}
-                                    </option
-                                    >
+                                        + perm | translate }}
```

Comment:
i think we can use something like this:
`data-ng-bind="::'pci_projects_project_training_jobs_submit_data_permission_'+ perm | translate"`